roles/logging/defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 logging:
+  version: 0.3.1
+  logstashforwarder_url: http://repo.openstack.blueboxgrid.com/ubuntu/pool/main/l/logstash-forwarder/logstash-forwarder_0.3.1_amd64.deb
```

Comment:
Is this reachable without access?  I think we should make this a default in roles/apt...no?